index.d.ts

```diff
@@ -4,6 +4,7 @@
  import https = require("https");
  import net = require("net");
 
+ http.createServer()
```

Comment:
This line looks out of place to me. Is this needed?

Reply:
Crap! Nice catch. Don't know how that got there. I guess I'm not as good with computer as I thought.